### PR TITLE
[SWA-97][FEAT] - Update sushiswap router and factory addresses to v3

### PR DESCRIPTION
**Pull request by ElRodrigote**
## Fixes: [SWA-97](https://linear.app/swaprhq/issue/SWA-97/update-sushiswap-to-v3)

# Description

* Update Sushiswap Factory and Router deployed contracts from V2 to V3

### CODE REVIEW

**Comment by Diogomartf** [comment]
We're publishing dist folder.

File: src/entities/trades/routable-platform/UniswapV2RoutablePlatform.ts [hunk 1]
```diff
@@ -20,13 +20,16 @@ const HONEYSWAP_FACTORY_ADDRESS = '0xA818b4F111Ccac7AA31D0BCc0806d64F2E0737D7'
 const LEVINSWAP_FACTORY_ADDRESS = '0x965769C9CeA8A7667246058504dcdcDb1E2975A5'
 const PANCAKESWAP_FACTORY_ADDRESS = { [ChainId.BSC_MAINNET]: '0xcA143Ce32Fe78f1f7019d7d551a6402fC5350c73' }
 const QUICKSWAP_FACTORY_ADDRESS = '0x5757371414417b8C6CAad45bAeF941aBc7d3Ab32'
+/**
+ * @see https://github.com/sushiswap/v3-periphery/tree/master/deployments
+ */
```
[the remark this answers is not in the document]
Comment (ElRodrigote) [reply src/entities/trades/routable-platform/UniswapV2RoutablePlatform.ts:25]: Sorry, didn't understand this one.

**Comment by ElRodrigote** [comment]
> We're publishing dist folder which usually is unintended.
> 
> If we want to have access to dist to test the SDK it should be on the description but would be better to have that on another PR.
> 
> So this one would be only the new updates and create a branch from this one with the dist folder if needed to test `SDK`.

Oh, I did it this way because that's how we usually do it with @berteotti , but I'm open to suggestions!

File: src/entities/trades/routable-platform/UniswapV2RoutablePlatform.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (Diogomartf) [reply src/entities/trades/routable-platform/UniswapV2RoutablePlatform.ts:25]: Why left the comment?

**Comment by ElRodrigote** [comment]
Closing this PR for now, the ticket changed to update Sushiswap V2 addresses. V3 support will be added in another issue.

**Pull request closed without merging**